toolkit/components/uniffi-bindgen-gecko-js/src/render/js.rs

```diff
@@ -7,9 +7,31 @@ use askama::Template;
 use extend::ext;
 use heck::{CamelCase, MixedCase, ShoutySnakeCase};
 use uniffi_bindgen::interface::{
-    Argument, ComponentInterface, Error, FFIFunction, Field, Function, Record, Type, Enum
+    Argument, ComponentInterface, Error, FFIFunction, Field, Function, Record, Type, Enum, Object, Constructor, Method, Literal, Radix
 };
 
+fn arg_names(args: &[&Argument]) -> String {
+    args.iter()
+        .map(|arg| {
+            if let Some(default_value) = arg.default_value() {
+                format!("{} = {}", arg.nm(), default_value.render())
```

Comment:
realized we were missing default value implementation and it was easy enough to add here so I added it